src/libs/SidebarUtils.js

```diff
@@ -259,6 +262,17 @@ function getOptionData(reportID) {
     if (result.isChatRoom || result.isPolicyExpenseChat) {
         result.alternateText = lastMessageText || subtitle;
     } else {
+        if (hasMultipleParticipants && !lastMessageText) {
```

Comment:
It's not super obvious what the code below is doing. Let's add a comment explaining it.

Reply:
Perfect, thanks!